### PR TITLE
update the metric for rgw_health_status

### PR DESCRIPTION
**Pull request by weirdwiz**
Include more conditions in RGW health status, and update the alerts based on the changes

### CODE REVIEW

**Comment by weirdwiz** [comment]
oh looks like i pushed a dirty branch
fixing

**Comment by weirdwiz** [comment]
ptal @umangachapagain

File: metrics/internal/collectors/ceph-object-store.go [hunk 1]
```diff
@@ -97,24 +97,48 @@ func getAllObjectStores(lister cephv1listers.CephObjectStoreLister, namespaces [
 func (c *CephObjectStoreCollector) collectObjectStoreHealth(cephObjectStores []*cephv1.CephObjectStore, ch chan<- prometheus.Metric) {
 	for _, cephObjectStore := range cephObjectStores {
 		switch cephObjectStore.Status.Phase {
-		case cephv1.ConditionConnected:
+		case cephv1.ConditionConnecting:
```
Comment (umangachapagain) [inline metrics/internal/collectors/ceph-object-store.go:100]: I don't think we should be changing existing values.

**Review by umangachapagain — APPROVED** [review]

**Pull request merged**